REDCapUITweaker.php

```diff
@@ -2811,10 +3083,10 @@ function provideStaticFormVarName()
         vFormLink.css('display', 'inline-block')
         var vSaveBtn = $(this).find( '#form_menu_save_btn-' + vFormName )
         var vChgName = $('<a href="#"><i class="fas fa-pencil-alt fs8"></i></a>')
-        vChgName.attr('title', 'Edit instrument variable name')
+        vChgName.attr('title', 'Edit form variable name')
```

Comment:
I thought generally accepted REDCap lingo was to call them 'instruments' and not 'forms' even though we tend to in NCTU.  I'm not too worried about this - but as it is a globally available module, I think it should follow the global standard and not ours.  If it matters much, which I guess it doesn't really!

Reply:
The latest version of REDCap has built-in functionality which partially overlaps with that provided by the static form names feature and I have amended the wording here to better align with what REDCap is using.